src/wp-admin/edit-form-blocks.php

```diff
@@ -126,9 +126,14 @@ static function ( $classes ) {
 // Set Heartbeat interval to 10 seconds, used to refresh post locks.
 wp_add_inline_script(
 	'heartbeat',
-	'if ( window.wp && window.wp.heartbeat ) {
-		window.wp.heartbeat.interval( 10 );
-	}'
+	'if ( window.wp && window.wp.domReady ) {
+		window.wp.domReady( function() {
+			if ( window.wp.heartbeat ) {
+				window.wp.heartbeat.interval( 10 );
+			}
+		} );
+	}',
+	'before'
```

Comment:
Changing this to `after` will negate the need to check heartbeat is defined, the script defines it as the script loads.

https://github.com/WordPress/wordpress-develop/blob/47122105f20624e5a6b8ed151d732157f2f2605c/src/js/_enqueues/wp/heartbeat.js#L892-L898